src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -22,16 +24,19 @@
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_TRIPDAY + "DAY "
+            + PREFIX_TRIPTIME + "TIME "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
+            + PREFIX_NAME + "Jenny Talia "
             + PREFIX_PHONE + "98765432 "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_ADDRESS + "369, Yishun Ave 4, #02-25 "
+            + PREFIX_TRIPDAY + "wednesday "
+            + PREFIX_TRIPTIME + "1930 "
+            + PREFIX_TAG + "female ";
```

Comment:
Should there be a space after `female`? Not an issue, just a formatting thing really